source/extensions/resource_monitors/cpu_utilization/linux_cpu_stats_reader.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include <string>
+
+#include "source/extensions/resource_monitors/cpu_utilization/cpu_stats_reader.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace ResourceMonitors {
+namespace CpuUtilizationMonitor {
+
+static const std::string LINUX_CPU_STATS_FILE = "/proc/stat";
+
+constexpr uint64_t NUMBER_OF_CPU_TIMES_TO_PARSE = 4;
+constexpr uint64_t MICROSECONDS = 1000 * 1000;
```

Comment:
These can go in the implementation file? Also can NUMER_OF_CPU_TIMES_TO_PARSE be commented? Why 4 times?

Reply:
Moved.